flexget/plugins/services/myepisodes.py

```diff
@@ -1,27 +1,25 @@
-from __future__ import unicode_literals, division, absolute_import
-from builtins import *  # pylint: disable=unused-import, redefined-builtin
```

Comment:
Please don't remove these two lines. They are quite important for Python 3 compatibility.